### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by azime**

### CODE REVIEW

File: analyzers/coverage_journeys_requests_params.py [hunk 1]
```diff
@@ -21,19 +21,6 @@ def get_tuples_from_stat_dict(stat_dict):
...
-        if rdd.count():
-            wheelchair_stats = rdd.flatMap(self.get_tuples_from_stat_dict) \
-                .reduceByKey(lambda a, b: a + b) \
-                .collect()
-            return [tuple(list(key_tuple) + [nb]) for (key_tuple, nb) in wheelchair_stats]
-        return []
-
-    def get_data(self):
-        files = self.get_files_to_analyze()
-        rdd = self.load_data(files, rdd_mode=True)
```
Comment (rchoquet) [inline analyzers/coverage_journeys_requests_params.py]: rdd_mode=True needs to be at L35

File: analyzers/coverage_journeys_transfers.py [hunk 2]
```diff
@@ -33,22 +33,6 @@ def get_tuples_from_stat_dict(stat_dict):
...
-            ).reduceByKey(
-                lambda a, b: (a + b)
-            ).collect()
-            return [tuple(list(key_tuple) + [nb]) for (key_tuple, nb) in data]
-        else:
-            return []
-
-    def get_data(self):
-        files = self.get_files_to_analyze()
-        rdd = self.load_data(files, rdd_mode=True)
```
Comment (rchoquet) [inline analyzers/coverage_journeys_transfers.py]: Same here i think

File: analyzers/coverage_journeys_requests_params.py (hunk 1 above)
Comment (azime) [inline analyzers/coverage_journeys_requests_params.py]: ok

File: analyzers/coverage_journeys_transfers.py (hunk 2 above)
Comment (azime) [inline analyzers/coverage_journeys_transfers.py]: ok

**Comment by nberard** [comment]
can u squash into 2 or 3 commits?

**Pull request merged**